components/dropbox/actions/upload-file/upload-file.mjs

```diff
@@ -0,0 +1,101 @@
+import dropbox from "../../dropbox.app.mjs";
+import consts from "../../consts.mjs";
+import fs from "fs";
+import got from "got";
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  name: "Upload a File",
+  description: "Uploads a file to a selected folder. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesUpload__anchor)",
+  key: "dropbox-upload-a-file",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dropbox,
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFolder",
+      ],
+      description: "The file path in the user's Dropbox to create the folder. If not filled, it will be created in the root folder.",
```

Comment:
```suggestion
      description: "The file path in the user's Dropbox to create the file. If not filled, it will be created in the root folder.",
```